packages/@react-native-windows/cli/src/config/configUtils.ts

```diff
@@ -412,3 +412,24 @@ export function getProjectNamespace(projectContents: Node): string | null {
 export function getProjectGuid(projectContents: Node): string | null {
   return tryFindPropertyValue(projectContents, 'ProjectGuid');
 }
+
+export function getExperimentalFeatures(
+  solutionDir: string,
+): Record<string, string> | null {
+  const propsFile = path.join(solutionDir, 'ExperimentalFeatures.props');
+  if (fs.existsSync(propsFile)) {
+    const result: Record<string, any> = {};
+    const propsContents = readProjectFile(propsFile);
+    const nodes = msbuildSelect(
+      `//msbuild:PropertyGroup/msbuild:*`,
+      propsContents,
+    );
+    for (const node of nodes) {
+      const propertyNode = node as Node;
+      result[propertyNode.nodeName] = propertyNode.textContent;
```

Comment:
Technically should we 'parse' if it is a boolean or an 'expression'?

Reply:
I thought about this and decided to just leave it as a string and let the end consumer decide what they want to do with it, since we have no real contract for what kinds of properties are allowed in experimentalfeatures.props.